src/pages/dev/amplifier/chain-integration/relay-messages.mdx

```diff
@@ -0,0 +1,157 @@
+```
+
+```bash
+axelard tx wasm execute $SOURCE_CHAIN_GATEWAY \
+    '{
+       "verify_messages": [
+            {
+                "cc_id": {
+                    "chain":"'"$CHAIN_NAME"'",
+                    "id":"'"$TX_HASH-$TX_EVENT"'"
+                },
+                "destination_chain":"'"$DESTINATION_CHAIN"'",
+                "destination_address":"'"$DESTINATION_ADDRESS"'",
+                "source_address":"'"$SOURCE_ADDRESS"'",
+                "payload_hash":"'"$PAYLOAD_HASH"'"
+            }
+       ]
+    }' \
+    --keyring-backend test  \
+    --from wallet  \
+    --gas auto --gas-adjustment 1.5 --gas-prices 0.007uverifiers  \
+    --chain-id=devnet-verifiers \
+    --node $RPC
+```
+
+## Route messages
+
+[`route_messages`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/packages/gateway-api/src/msg.rs#L14) is a command on the gateway that routes successfully verified messages to the destination chain. It takes a vector of verified messages to be routed.
+
+```bash
+SOURCE_CHAIN_GATEWAY="axelar1vnfn0e4vnn58ydpm05wqcc9zp8t5ztwd5rw5f895lykqaezmuccqujmwl2"
+export CHAIN_NAME="avalanche"
+export TX_HASH="7C7C74495FFD6CCC6B4B5E7069CF2495485F59E9CA5E8222DA486CE08DADC7E8"
+export TX_EVENT="0"
+export DESTINATION_CHAIN="ethereum-sepolia"
+export DESTINATION_ADDRESS="0x8f8dedd09E23E22E1555e9D2C25D7c7332291919"
+export SOURCE_ADDRESS="0x0a3b8dc7706c47b6dd87d771df63875b1c5cd867"
+export PAYLOAD_HASH="220f68445e3cec114bff50cd6b251e3deabc7684b10280c2116b20bcc6795a96"
+export RPC="http://devnet-verifiers.axelar.dev:26657"
+```
+
+```bash
+axelard tx wasm execute $SOURCE_CHAIN_GATEWAY \
+    '{
+       "route_messages":[
+            {
+                "cc_id": {
+                    "chain":"'"$CHAIN_NAME"'",
+                    "id":"'"$TX_HASH-$TX_EVENT"'"
+                },
+                "destination_chain":"'"$DESTINATION_CHAIN"'",
+                "destination_address":"'"$DESTINATION_ADDRESS"'",
+                "source_address":"'"$SOURCE_ADDRESS"'",
+                "payload_hash":"'"$PAYLOAD_HASH"'"
+            }
+       ]
+    }' \
+    --keyring-backend test  \
+    --from wallet  \
+    --gas auto --gas-adjustment 1.5 --gas-prices 0.007uverifiers  \
+    --chain-id=devnet-verifiers \
+    --node $RPC
+```
+
+## Construct a proof
+
+[`construct_proof`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/contracts/multisig-prover/src/msg.rs#L61) takes a vector of `CrossChainID`s and builds a proof that includes the messages to be routed so that the message can be relayed to the destination gateway. It also passes the message from the gateway to the prover.
+
+```bash
+export DESTINATION_CHAIN_MULTISIG_PROVER="axelar1xz4cya4qm2ws6nzperhvc40wdjcq4872fl6d3j2s4cytyx8j80eqenv87g"
+export CHAIN_NAME="avalanche"
+export TX_HASH="0x41d08bd627d7be301e0858312b63b3ede840d7fad145a9da333c26adf5d98614"
```

Comment:
Should we make the assumption that the user is going to run all of these commands and not repeat the variables that shouldn't change? We do this in the integrators guide, assume step 1 has been executed (and we have all those vars) for step 2.

Reply:
Sounds good to me. I was going back and forth on this for a bit because it might be good to see every param within the context of the same code sample, but for length's sake we can trim it down.